app/components/team_sidebar/team_list/team_item/team_icon.tsx

```diff
@@ -41,8 +42,9 @@ export default function TeamIcon({id, lastIconUpdate, displayName, selected}: Pr
         teamIconContent = (
             <Text
                 style={styles.text}
+                testID={`${testID}.display_name_abbreviation`}
             >
-                {displayName?.substr(0, 2).toUpperCase()}
+                {displayName?.substring(0, 2).toUpperCase()}
```

Comment:
I wonder why it has to change.

Reply:
@saturninoabril substr looks deprecated, so I changed to the updated function.

Reply:
Oh, I see - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr. Thanks!

Reply:
Btw, we have it in several places in mobile and webapp. So maybe a good separate fix/campaign and/or announcement to Dev channel on how they'd like to update? I'm good as is but will defer to Mobile team.

Reply:
@saturninoabril I'm afraid I don't understand what you mean here

Reply:
are you referring to the deprecated `substr` function ? is been deprecated for years.. I see it being used in 8 other places, so I don't think this needs a campaign, only one PR should do it, although is still supported by the JS engine used in mobile

Reply:
Yes, `substr`. Sorry, I meant a separate fix for mobile for consistency (in one PR) then on webapp as well, and I mentioned "announcement" to remind everyone not to use for new code. Yup, I can see it's still supported that's why I'll defer to dev on how you/they want to update.
Nothing blocking from me. I just made a comment since I don't know if such change has anything to do with this Detox/E2E PR but I could be totally wrong.